scripts/blocks-utils.js

```diff
@@ -104,10 +106,22 @@ function observe(elementToObserve, callback, ...args) {
   observer.observe(elementToObserve);
 }
 
+function getOriginUrl() {
+  return WORKER_ORIGIN_URL;
+}
+
+function getResearchAPIUrl() {
+  return `${getOriginUrl()}/CDNResearchAPI/CallResearchAPI`;
```

Comment:
can't we have constants for these instead of methods? 
RESEARCH_API_URL 
MARKETING_API_URL